widget/form_test.go

```diff
@@ -121,7 +121,7 @@ func TestForm_ChangeTheme(t *testing.T) {
 		},
 		OnSubmit: func() {}, OnCancel: func() {}}
 	w := test.NewWindow(form)
-	w.Resize(fyne.NewSize(340, 240))
+	w.Resize(fyne.NewSize(340, 280))
```

Comment:
Could this use `form.MinSize()` so the golden images are as small as possible?

Reply:
To my knowledge in the theme change tests we have not done so, because the min-size changes mid-test.
The number there is actually smaller than the minimum on the ugly test, as can be seen by the thin bottom edge.

Reply:
The window will resize to accommodate the content's min size tho, and the default and ugly themes are compared against separate images so their sizes don't need to match. I bring it up because it there was concern image tests were too large, this is one way to reduce them

Reply:
Yeah fair point. Maybe we should make a mental note to fix each as we see them.
I've updated these ones now.